### PR TITLE
Review one of my past projects

### PR DESCRIPTION
**Pull request by KDlamini**
In this pull request, I will be reviewing one of the past projects that I did during the Microverse Admissions trials. I  must check for the following requirements.
- Apply best practices in HTML code.
- Use semantic HTML tags.
- Use CSS selectors correctly.
- Use CSS box model.
- Make sure that there are no linter errors.
- Make sure that the project is documented in a professional way.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,8 +64,8 @@ <h3>POWERS</h3>
       </section>
 
       <footer>
-        <p class="ref">All information in this page is derived from the <a href="https://disney.fandom.com/wiki/Thor_Odinson">Disney Fandom</a> website.</p>
-        <a href="https://en.wikipedia.org/wiki/Thor_(film)" id="tribute-link" target="_blank"> Learn More</a>
+        <p class="ref">All information in this page is derived from the <a href="https://disney.fandom.com/wiki/Thor_Odinson" rel=”noopener”>Disney Fandom</a> website.</p>
+        <a href="https://en.wikipedia.org/wiki/Thor_(film)" id="tribute-link" rel=”noopener” target="_blank"> Learn More</a>
```
Comment (KDlamini) [inline index.html]: Please add a rel="nonreferrer" attribute to the link elements to fix your lint errors.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,43 @@
...
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- This project is part of week 2 of module 1 in the Microverse study program.
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (KDlamini) [inline README.md:43]: Please add your MIT.md file to your project and make sure that it is modified to fit the project.

**Review by KDlamini — COMMENTED** [review]
# Changes Required ♻️ 

Hey There! 
I'm Simo, I am your reviewer for your PR.

Great job on completing your project. You are almost there. There are a few changes you need to make to move forward.
Please make the changes required in the comments.
Be sure to check your lint errors and make sure that they all pass.

Submit for a re-review when you are done.
Thanks 😁 .

**Review by mimipeshy — APPROVED** [review]
## Hi @KDlamini 👋🏾 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**